src/rio.h

```diff
@@ -81,6 +81,7 @@ struct _rio {
             FILE *fp;
             off_t buffered; /* Bytes written since last fsync. */
             off_t autosync; /* fsync after 'autosync' bytes written. */
+            unsigned reclaim_cache:1; /* A flag to indicate reclaim cache after fsync */
```

Comment:
i think we better explicitly set it to 0 in rioInitWithFile, otherwise, i'm not sure we can be certain what't the value.